### DISCUSSION TITLE
Three- Four-Five

### DISCUSSION THREAD

**Discussion by ghost** [systemphil/sphil#125] [Hegel]
Hello

As we know, Hegel says that the dialectical method is trichotomous but contains implicitly a fourth moment.

> If one insists on counting, this second immediate [i.e., the negation of the negation] is, in the course of the method as a whole, the third term to the first immediate and the mediated. It is also, however, the third term to the first or formal negative and to absolute negativity or the second negative; now as the first negative is already the second term, the term reckoned as third can also be reckoned as fourth, and instead of a triplicity, the abstract form may be taken as a quadruplicity; in this way, the negative or the difference is counted as a duality."

This quadruplicity tends to affirm itself in the philosophy of nature, which is entirely structured on the Four Elements, deriving themselves from the four kinds of cosmic bodies (sun, comet, moon, planet).

Now I have had this strange idea : is it possible that there be also a fivefold movement in the dialectic ? And that this Five would tend to express itself in the Philosophy of Spirit ?

But the expression of the Five would be more subtle than the Fourth’s, since the philosophy of spirit is returning to the « purity » of the triangular Logic. This Five could be expressed as a kind of « quantum superposition » of states. For instance, when Hegel describes, on the logical plane, three forms of Art : symbolic, classic and romantic. The fivefold logic would be a logical step too, but one happening on a subtly different level, saying that these 3 « states of the art » can be mixed, and that there exists 2 canonical results for this mixing up, setting the count to 5.

There exists a paradigmatic example for this : the greek architectural orders are three : doric, ionic and corinthian (that, in this order, more or less replicate the symbolic-classic-romantic scheme). But the latins added two other orders : the tuscan and the composite. The tuscan is very sober, and could be thought of as a minimum, a ground zero for each and every column. On the contrary, the composite is very heavy, it is a baroque maximum, a saturated form of art. Analogically, if doric-corinthian-ionic were the three primary colors, then, the tuscan and the composite would be like white and black (rather than secondary colors in the usual sense).

If the additional orders of architecture were secondary colors, we should rather expect to have 6 orders in toto (mixing them two by two). But the fivefold structure disclosed by History indicates that they are rather mixed by the ALL and NONE logical operators.

And so I am making this bold hypothesis : could we expect to find these two ALL-and-NONE-induced additional categories, during the philosophy of spirit (at a secondary, more subtle logical level though).

**Comment by Firgrep** [comment]
It's an interesting thought. And why stop at five, why not six, seven or thirteen? It's fun to find examples around in the _Realphilosophie_, but I think the real challenge will be to demonstrate these patterns in the _Logic_. If it's not there in the metaphysics, I don't think we'll find it anywhere else. 

I think any new patterns, movements or N-folded structure moments need to be shown how they:
- become explicit logically
- are immanently derived
- help explain the content better than previously
- help explain the content more with less (this is a tough one)

Do you think we can find any such in the _Logic_? 

There are definitely new movements, moments and patterns we can find the metaphysics I think!

**Comment by ghost** [comment]
I agree with your « conditions »

why stop at five ? Because the intuition is that there is only one n-fold that matches your conditions in any big section of the Encyclopedia : Logic, Nature, Spirit. So that we could not go higher than 5 in any structural way.

to be honest, the sequence 3-4-5 is a bit mystical, it represents both Pî (3,1415) and the so-called Egyptian Triangle. I find this special combination of a rectangular triangle and a circle particularly interesting. It seems natural that the dialectical dancing step itself should follow a dialectic (this reflexion of categories is very hegelian). When reaching a fivefold rhythm, one would actually return to a kind of triangle, and a rectangular one on top of that, showing how Spirit has « conserved » the preceding moments.
